src/puppetlabs/puppetdb/config.clj

```diff
@@ -318,8 +318,6 @@
   "Configures the global properties from the user defined config"
   [{:keys [global] :as config}]
   (let [product-name (normalize-product-name (get global :product-name "puppetdb"))]
-    (when (:event-query-limit global)
```

Comment:
Since global isn't used by multiple things now it would be more concise to make the params simply `config` and the above `get` a `get-in config [:global :product-name]`